### DISCUSSION TITLE
请教面板里面显示ip都是127

### DISCUSSION THREAD

**Discussion by Daohuai0913** [kkkgo/PaoPaoGateWay#110] [问和答]
![1](https://github.com/user-attachments/assets/6d00f262-c29c-48ca-8659-2c6c56fb4a9d)
![2](https://github.com/user-attachments/assets/0db464ed-e8e7-4bc9-b06c-b51163f27ecd)
如图 在ikuai nat规则里面也是设置了 过滤fakeIP段的nat 结果全是127 是我哪里设置的不对吗？

**Comment by jaesony** [comment]
是不是用的定制镜像加了sniff=yes
